Header header
bool ball_detected
float64 ball_range
float64 ball_bearing
float64 ball_radius
int64 ball_center_x
int64 ball_center_y
float64 kick_time
